### ISSUE TITLE
Cannot use AppleScript for shortcuts containing arrows in Sierra, Works in El Capitan

### ISSUE THREAD

**Issue by xengravity** [eczarny/spectacle#753]
Just upgraded from El Capitan to Sierra. I have AppleScripts created for Spectacle shortcuts that stopped working.

Through testing, I have found that all shortcuts containing arrow keys do not work.

In Spectacle, setting "Left Half" = control + option + command + left arrow with the following Apple Script no longer works:

tell application "System Events"
	keystroke 123 using {control down, option down, command down}
end tell

But doing the same with "Center" = control + option + command + "c" works with AppleScript:

tell application "System Events"
	keystroke "c" using {control down, option down, command down}
end tell

**Comment by fobin** [comment]
I think it isn't just Sierra but specifically 10.12.4 which broke this. It worked fine before that.

**Comment by pinjaliina** [comment]
According to my tests this remains unfixed in 10.12.6 with the latest updates installed, but is fixed in 10.13.2. (My Applescript was unrelated to Spectacle.)

**Comment by Didericis** [comment]
I had this issue in 10.14.4 and fixed it by using `key code` instead of `keystroke`.

**Doesn't work**
```applescript
tell application "System Events"
	keystroke 123 using {control down, command down}
end tell
```

**Works**
```applescript
tell application "System Events"
	key code 123 using {control down, command down}
end tell
```
